config/config-example.js

```diff
@@ -96,6 +96,14 @@ exports.pmmodchat = false;
 //   This setting can also be turned on with the command /forcetimer.
 exports.forcetimer = false;
 
+// auto save replays - automatically attempt to send replays after a battle ends
+//   Useful for tournament servers so that users don't have to waste time saving
+//   the battles manually. Server must be registered.
+//
+// Your server *must* be registered in order to save replays, so don't turn this
+// on unless it is registered!
```

Comment:
Maybe add a bunch of warnings here,

or just leave it out of config-example completely and make it clear it's undocumented and not designed for public consumption